spec/abstractops.html

```diff
@@ -29,6 +29,116 @@ <h1>IterableToListOfType ( _items_, _elementTypes_ )</h1>
+          = 1 if 31 ≤ DayWithinYear(_t_) &lt; 59 + InLeapYear(_t_)
+          = 2 if 59 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 90 + InLeapYear(_t_)
+          = 3 if 90 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 120 + InLeapYear(_t_)
+          = 4 if 120 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 151 + InLeapYear(_t_)
+          = 5 if 151 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 181 + InLeapYear(_t_)
+          = 6 if 181 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 212 + InLeapYear(_t_)
+          = 7 if 212 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 243 + InLeapYear(_t_)
+          = 8 if 243 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 273 + InLeapYear(_t_)
+          = 9 if 273 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 304 + InLeapYear(_t_)
+          = 10 if 304 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 334 + InLeapYear(_t_)
+          = 11 if 334 + InLeapYear(_t_) ≤ DayWithinYear(_t_) &lt; 365 + InLeapYear(_t_)
+      </emu-eqn>
+      <p>where</p>
+      <emu-eqn id="eqn-DayWithinYear" aoid="DayWithinYear">DayWithinYear(_t_) = ℝ(Day(_t_)) - DayFromYear(YearFromTime(_t_))</emu-eqn>
+      <p>A month value of 0 specifies January; 1 specifies February; 2 specifies March; 3 specifies April; 4 specifies May; 5 specifies June; 6 specifies July; 7 specifies August; 8 specifies September; 9 specifies October; 10 specifies November; and 11 specifies December. Note that <emu-eqn>MonthFromTime(0) = 0</emu-eqn>, corresponding to Thursday, 1 January 1970.</p>
+      <emu-clause id="sec-date-number">
+        <h1>Date Number</h1>
+        <p>A date number is identified by a mathematical value in the inclusive interval from 1 to 31. The mapping DateFromTime(_t_) from a time value _t_ to a date number is defined by:</p>
+        <emu-eqn aoid="DateFromTime">
+          DateFromTime(_t_)
+            = DayWithinYear(_t_) + 1 if MonthFromTime(_t_) is 0
+            = DayWithinYear(_t_) - 30 if MonthFromTime(_t_) is 1
+            = DayWithinYear(_t_) - 58 - InLeapYear(_t_) if MonthFromTime(_t_) is 2
+            = DayWithinYear(_t_) - 89 - InLeapYear(_t_) if MonthFromTime(_t_) is 3
+            = DayWithinYear(_t_) - 119 - InLeapYear(_t_) if MonthFromTime(_t_) is 4
+            = DayWithinYear(_t_) - 150 - InLeapYear(_t_) if MonthFromTime(_t_) is 5
+            = DayWithinYear(_t_) - 180 - InLeapYear(_t_) if MonthFromTime(_t_) is 6
+            = DayWithinYear(_t_) - 211 - InLeapYear(_t_) if MonthFromTime(_t_) is 7
+            = DayWithinYear(_t_) - 242 - InLeapYear(_t_) if MonthFromTime(_t_) is 8
+            = DayWithinYear(_t_) - 272 - InLeapYear(_t_) if MonthFromTime(_t_) is 9
+            = DayWithinYear(_t_) - 303 - InLeapYear(_t_) if MonthFromTime(_t_) is 10
+            = DayWithinYear(_t_) - 333 - InLeapYear(_t_) if MonthFromTime(_t_) is 11
+        </emu-eqn>
+      </emu-clause>
+      <emu-clause id="sec-week-day">
+        <h1>Week Day</h1>
+        <p>The weekday for a particular time value _t_ is defined as</p>
```

Comment:
We should not mention "time value" in these descriptions, since that concept is specific to legacy Date and is defined as a JS Number with certain limits and including NaN, not a mathematical value.